setup.py

```diff
@@ -25,7 +25,7 @@
     lambda: check_CPP14_flags(["-std=c++14", "-std=c++1y", "-std=gnu++0x"]),
     {"msvc": ["/std:c++14"]},
 )
-CPP03_CONFIG = defaultdict(lambda: ["-std=c++03"], {"msvc": ["/std:c++03"]})
+CPP11_CONFIG = defaultdict(lambda: ["-std=c++11"], {"msvc": ["/std:c++11"]})
```

Comment:
I agree, this seems the right thing to do.  Now that EWAH has been split out, do we still use C++14 anywhere?

Reply:
Ewah has been split out ?

Reply:
ignore me, I am all gutters no strikes today.